docs/diagnostics/PX1007.md

```diff
@@ -8,18 +8,20 @@ This document describes the PX1007 diagnostic.
 | PX1007 | A public entity or DAC property should have a description in the `summary` XML tag. | Warning | Available | 
 
 ## Diagnostic Description
-Any public class, interface, structure, enumeration, delegate, or their member that is defined in a `PX` namespace (such as the `PX.Objects.AR` namespace) should have a description in the `summary` XML tag. These descriptions can be used for building the API and DAC references for the Acumatica ERP libraries.
+Any public class, interface, struct, enumeration, delegate, or their member that is defined in a `PX` namespace (such as the `PX.Objects.AR` namespace) should have a description in the `summary` XML tag. These descriptions can be used for building the API and DAC references for the Acumatica ERP libraries.
```

Comment:
We removed the logic checking the namespace. It works for any namespace now

Reply:
This is a description of the requirement. We need only the entities from PX namespaces to be documented.
It is written below that the diagnostic works for any namespace.

Reply:
I can't see the description. For me it looks a bit confusing. For example, I have an alert for class in namespace `MyNamespace`, open the diagnostic description from VS and see that
> Any public class, interface, struct, enumeration, delegate, or their member that is defined in a `PX`
I'm confused now because my class is obviously not defined there